src/test/java/co/rsk/federate/signing/hsm/message/AdvanceBlockchainMessageTest.java

```diff
@@ -129,26 +130,26 @@ public void test_getParsedBrothers_with_more_than_10_brothers() throws HSMBlockc
         BlockHeader blockHeader3 = blockHeaderBuilder.setNumber(3).build();
 
         List<BlockHeader> block1Brothers = Arrays.asList(
-            blockHeaderBuilder.setNumber(101).build(),
-            blockHeaderBuilder.setNumber(102).build(),
-            blockHeaderBuilder.setNumber(103).build(),
-            blockHeaderBuilder.setNumber(104).build(),
-            blockHeaderBuilder.setNumber(105).build(),
-            blockHeaderBuilder.setNumber(106).build(),
-            blockHeaderBuilder.setNumber(107).build(),
-            blockHeaderBuilder.setNumber(108).build(),
-            blockHeaderBuilder.setNumber(109).build(),
-            blockHeaderBuilder.setNumber(110).build(),
-            blockHeaderBuilder.setNumber(111).build(),
-            blockHeaderBuilder.setNumber(112).build(),
-            blockHeaderBuilder.setNumber(113).build()
+            blockHeaderBuilder.setNumber(101).setDifficulty(new BlockDifficulty(BigInteger.valueOf(24))).build(),
+            blockHeaderBuilder.setNumber(102).setDifficulty(new BlockDifficulty(BigInteger.valueOf(42))).build(),
+            blockHeaderBuilder.setNumber(103).setDifficulty(BlockDifficulty.ONE).build(),
+            blockHeaderBuilder.setNumber(104).setDifficulty(new BlockDifficulty(BigInteger.valueOf(27))).build(),
+            blockHeaderBuilder.setNumber(105).setDifficulty(new BlockDifficulty(BigInteger.valueOf(99))).build(),
+            blockHeaderBuilder.setNumber(106).setDifficulty(new BlockDifficulty(BigInteger.valueOf(55))).build(),
+            blockHeaderBuilder.setNumber(107).setDifficulty(new BlockDifficulty(BigInteger.TEN)).build(),
+            blockHeaderBuilder.setNumber(108).setDifficulty(new BlockDifficulty(BigInteger.valueOf(32))).build(),
+            blockHeaderBuilder.setNumber(109).setDifficulty(new BlockDifficulty(BigInteger.valueOf(39))).build(),
+            blockHeaderBuilder.setNumber(110).setDifficulty(new BlockDifficulty(BigInteger.valueOf(65))).build(),
+            blockHeaderBuilder.setNumber(111).setDifficulty(new BlockDifficulty(BigInteger.valueOf(14))).build(),
+            blockHeaderBuilder.setNumber(112).setDifficulty(new BlockDifficulty(BigInteger.valueOf(43))).build(),
+            blockHeaderBuilder.setNumber(113).setDifficulty(new BlockDifficulty(BigInteger.valueOf(80))).build()
```

Comment:
Just an idea, to make it easier to follow.

Have 10 brother with difficulty between 100 and 110 so it's easy to tell that those are the ones to be selected. For the rest, put a difficulty between 10 and 20. 

What do you think?

Reply:
Yes, It Better Like You Suggested. Updated